crypto/crypto/src/hkdf.rs

```diff
@@ -152,6 +152,7 @@ where
 /// c) small PRK value in HKDF-Expand according to RFC 5869.
 /// d) any other underlying HMAC error.
 #[derive(Clone, Debug, PartialEq, Eq, Error)]
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
Is this needed since you whitelisted this in x.toml?

Reply:
It's not, will remove.

Reply:
Done